packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts

```diff
@@ -296,19 +296,19 @@ test(testCase, async (t: Test) => {
         fake: 4,
       };
       await apiClient.invokeContractV1(
-        (parameters as any) as InvokeContractV1Request,
+        (parameters as unknown) as InvokeContractV1Request,
       );
     } catch (e) {
       t2.equal(
         e.response.status,
         400,
         `Endpoint ${fInvoke} with fake=4: response.status === 400 OK`,
       );
-      const fields = e.response.data.map((param: any) =>
+      const fields = e.response.data.map((param: { path: string }) =>
         param.path.replace(".body.", ""),
       );
       t2.ok(
-        fields.includes("fake"),
+        fields?.includes("fake"),
```

Comment:
@micoferdinand98 I think the question mark is not needed anymore, could you please double check?